features/cellular/framework/targets/UBLOX/AT/UBLOX_AT_CellularStack.cpp

```diff
@@ -404,36 +404,95 @@ void UBLOX_AT_CellularStack::clear_socket(CellularSocket *socket)
     }
 }
 
+#ifndef UBX_MDM_SARA_R41XM
 const char *UBLOX_AT_CellularStack::get_ip_address()
```

Comment:
Would it be possible to call get_ip_address(SocketAddress *) function here instead of duplicating the code? 
Would be much easier to remove later on.

Reply:
Something like
`SocketAddress addr;
get_ip_address(&addr);
return _ip; //Set already in above function`